### DISCUSSION TITLE
long delay between video and voice during a video call

### DISCUSSION THREAD

**Discussion by 4tler** [nextcloud/all-in-one#2056] [Questions] [answered]
Hello everyone, I switched to NC AIO some time ago and after a few teething problems I am now very satisfied. 

But what I have noticed from the beginning is the long delay between video and voice during a video call with Talk. I have more than a second between lip movement and the sound I hear.

Are there any adjustments that can be made or other ideas for improving this?

**Comment by szaimen** [comment, marked as answer]
Hi, we made a few improvements to the talk container with the v4.5.0 release. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
